### PR TITLE
Issue 2 user can view entries

### PR DESCRIPTION
**Pull request by macyhongg**
Kapture 2022-07-14 at 22 45 14

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -20,8 +51,49 @@ function logSubmit(event) {
     entryId: data.nextEntryId
   };
   data.nextEntryId++;
-  data.entries.unshift(entry);
+  entries.unshift(entry);
```
Comment (BrettAlbright) [inline js/main.js:55]: one thing to add is that inside of this function, you need to render the new entry and use the prepend method to add it to the DOM, so when the view switches to the entries view, you can see the newest entry.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking great! One update needed! Let me know if you have questions

File: js/main.js (hunk 1 above)
Comment (macyhongg) [reply js/main.js:55]: I have both of these things happening in the event listener when DOM content is loaded (main.js lines 93-99). Do I have to delete that and place it inside this function instead?

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**